balrogscript/src/balrogscript/submitter/cli.py

```diff
@@ -538,3 +538,49 @@ def __init__(self, api_root, auth0_secrets=None):
     def run(self, productName, version, build_number):
         name = get_release_blob_name(productName, version, build_number)
         ReleaseState(name, api_root=self.api_root, auth0_secrets=self.auth0_secrets).set_readonly()
+
+
+class SystemAddonsReleaseCreator(object):
+    def __init__(self, api_root, auth0_secrets):
+        self.auth0_secrets = auth0_secrets
+        self.api_root = api_root
+        self.url = "{}/{}".format(self.api_root, "releases")
```

Comment:
I think this means we'll be using v1 releases for now, which is the right call, since we want to be compatible with morgoth-cli as long as we're still using it.

Reply:
It does. I kept the behavior the same as morgoth-cli